index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Stock Exchange Project AV</title>
+    <link href="./css/style.css" rel="stylesheet" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <script src="./js/shared.js" defer></script>
+    <script src="./js/search-form.js" defer></script>
+    <script src="./js/search-results.js" defer></script>
+    <script src="./js/marquee.js" defer></script>
+    <script src="./js/index.js" defer></script>
+  </head>
+  <body>
+    <div class="marquee"></div>
+
+    <form id="search-form">
+      <div class="search-container">
+        <h2>Search Nasdaq Stocks</h2>
+        <div class="input-container">
+          <input
+            type="text"
```

Comment:
you should handle the "submit" event, in order to create better User Experience. If I press Enter when the input ios focused, the page refreshes.